test/wallet/createVirtualIban.js

```diff
@@ -0,0 +1,85 @@
+const { expect } = require('chai');
+const Chance = require('chance');
+const moment = require('moment');
+
+const Lemonway = require('../../');
+
+const chance = new Chance();
+
+describe('createVirtualIban', function () {
+  this.timeout(2000000);
+
+  let lemonway;
+  let walletOptions;
+  let wallet;
+
+  before(async() => {
+    lemonway = new Lemonway(process.env.LOGIN, process.env.PASS, process.env.ENDPOINT);
+    walletOptions = {
+      id: chance.word({ syllables: 5 }),
+      email: chance.email(),
+      firstName: chance.first(),
+      lastName: chance.last(),
+      country: 'FRA',
+      birthdate: moment().format('DD/MM/YYYY'),
+      isCompany: false,
+      nationality: 'FRA',
+      payerOrBeneficiary: true,
+    };
+
+    wallet = await lemonway.Wallet.create(chance.ip(), walletOptions);
+  });
+
+  it('should create a virtual iban', async() => {
+    const lemonwayIban = await wallet.createVirtualIBAN(chance.ip(), {
+      wallet: wallet.id,
+      country: 'FR'
+    });
+
+    expect(lemonwayIban).to.have.keys(['id', 'iban', 'bic', 'holder', 'domiciliation', 'status', 'maxAvailableIbanPerWalletLeft', 'maxAvailableIbanInTotalLeft']);
+
+    const updatedWallet = await lemonway.Wallet.getWalletDetails(chance.ip(), wallet);
+    expect(updatedWallet.wallet.ibans.length).to.eql(1);
+    expect(updatedWallet.wallet.ibans[0].iban).to.eql(lemonwayIban.iban);
```

Comment:
Wow but ain't the ibans for performing money outs also stated in this array ?

How would we differentiate them in such case ?

Reply:
Nice catch. Indeed we are doing 
```
if (_.get(data, 'IBANS')) {
  this.ibans = _.map(_.get(data, 'IBANS.IBAN', _.get(data, 'IBANS')), function (iban) {
    return new IBAN(iban);
  });
}
```
I will check maybe we could create two arrays 👍

Reply:
I've tested and when fetching the wallet details using `get` or `getWalletDetails`, lemonway returns all ibans in `IBANS`. 
```
IBANS LEMONWAY PACKAGE {
  IBAN: [
    {
      ID: '63494',
      S: '5',
      DATA: 'FR3530003000406794775792O03',
      SWIFT: 'FAKEBICXXX',
      HOLDER: 'FR Business'
    },
    {
      ID: '63495',
      S: '5',
      DATA: 'FR7630004029990001028202793',
      SWIFT: 'BNPAFRPPXXX',
      HOLDER: 'LEMONWAY'
    }
  ]
}
``` 
Only way to differentiate would be to use `HOLDER` but not sure we should do that. Let's have all ibans in the same array and let users to differentiate them when using the package.

Reply:
That's pretty sad but we cannot do much about it indeed.